PolyEngine/Engine/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ GenerateSourceGoups("${POLYENGINE_SRCS}")
 
 add_library(${ENGINE_TARGET} SHARED ${POLYENGINE_SRCS})
 target_compile_definitions(${ENGINE_TARGET} PRIVATE _ENGINE)
-target_include_directories(${ENGINE_TARGET} PUBLIC ${POLYENGINE_INCLUDE} ${BULLET_INCLUDE_DIRS} ${STB_INCLUDE_DIRS} ${IMGUI_INCLUDE_DIRS})
+target_include_directories(${ENGINE_TARGET} PUBLIC ${POLYENGINE_INCLUDE} ${BULLET_INCLUDE_DIRS} ${STB_INCLUDE_DIRS} "${ENGINE_ROOT_DIR}/ThirdParty/Imgui")
```

Comment:
Let's make the location of the imgui headers a variable, set in CMakeListsCommon.txt, just near the line with add_subdirectory.

Reply:
done